Makefile.am

```diff
@@ -261,13 +262,19 @@ BATS = \
 	test/functional/verify/verify-picky-whitelist.bats \
 	test/functional/verify/verify-skip-scripts.bats
 
+UNIT_TESTS = \
+	test/unit/test_strings.test
```

Comment:
This is not necessary for this PR, but I would really like a way of discovering these in the future. It is something I've been meaning to do for functional tests too so we don't have to add/remove every test to/from the list every time.

Reply:
+1 for this too, but I don't know how to do it. haha